### ISSUE TITLE
Alconost Video Contract

### ISSUE THREAD

**Issue by LaurenBuchman** [observablehq/.github#24]

**Comment by LaurenBuchman** [comment]
[NDA](https://docs.google.com/document/d/1pf3Q6c664WYHPpYIsOoSVCPl6349TFGAFRfpwKRZuv0/edit) has been sent over.
Next step will be to fill out a project brief, then we'll receive an estimate and a SOW from them.

**Comment by LaurenBuchman** [comment]
Email from Vendor:

I’d like to provide a summary of our conversation, examples and pricing for video production. Also, I’ve added a link to a Product Brief document below.

We produce different kinds of videos and provide full control over the process to our customers. In a nutshell, we create a script, voiceover, storyboard, graphics, animation, and music. We get your approval at each step of the way so that you know exactly what the final result will be like.

Regarding the price of videos, it depends on the number of scenes, overall complexity and provided materials (art sources) that we can use. 

A typical price range for certain types of videos:

Prerolls - ads videos you see on YouTube and Facebook/Instagram (5 - 15 seconds). [You can find a spreadsheet with prices here.](https://drive.google.com/file/d/1Scx5lB01Zcq8mqCVHhp-bGavrtuwFK5t/view?usp=sharing) It’s much cheaper to produce several at once. [Prerolls examples.](https://www.youtube.com/playlist?list=PLEYQSyvhM4CksuND84Yw4UvMGbZNapzq0)

Animated logo - $450 - $900 (up to 15 sec. video)

Video tutorials - $800 - $1200 (1-2 minutes videos). [Tutorials we produced for our customers](https://www.youtube.com/playlist?list=PLEYQSyvhM4CmOxBcmyEvchcrm3UDyPIjt).

AppStore/Google Play videos - $900 - $1800 based on customer requirements.

Videos with complex graphics and a large number of scenes like trailers, explanatory, how-to, etc. - $1700 - $2700 (45 - 90 seconds). The price can be higher/lower based on the complexity and specific requirements.
Examples:
[Celly.AI video](https://www.youtube.com/watch?v=I5ngUwjEX3s&t=8s)
[Bitrix24 video](https://www.youtube.com/watch?v=caxfW8BqQvM&t=1s)
[Jitsu video](https://www.youtube.com/watch?v=59cVVUxXxFU)
[Kaspersky Small Office Security video](https://www.youtube.com/watch?v=5ukkg2fYRLQ&t=44s)

3D videos - the price is calculated when the number of 3D models to create and the total complexity/length are known. Highly individual each time.

[You can find a list of our narrators here](https://soundcloud.com/alconost/sets/alconost-narrators/s-WP8H8) and choose the one you like for the video.

[Please fill out a short Product Brief document by following this link and I’ll get back to you with the project estimation from our production team](https://docs.google.com/document/d/1_mBUtVkqUi0WFqI-sERR3_0obUKsz1YbWWznkmOL63E/edit?usp=sharing).

Our docs templates:
[Contract template](https://docs.google.com/document/d/1gkVRhyDLBItAl4ok1RhxNRti2N_SBK976uEu5_3q9NU/edit?usp=sharing)
[NDA](https://docs.google.com/document/d/1_4ImnCcWMUcBsCbCzDnDzFwXOCZ47SXPaiDMnNVKjGs/edit?usp=sharing)

**Comment by LaurenBuchman** [comment]
@mariafciampa once we receive the NDA, we save a copy to the MarCom Vendor Operations Folder. I've created a subfolder for Alconost [here](https://drive.google.com/drive/u/0/folders/14OHlhfRfyqadr_hbf6eR05JnA2xdA_fn). We also save any SOWs in the same folder. It helps us stay organized.

**Comment by VPoltrack** [comment]
I'm currently working on storyboarding out the title and end cards, as well as the assets Maria shared (thank you Maria!) 
When those are ready, would love to grab 30 minutes with both of you so I can have you approve / give feedback. 
I'll keep you both updated

**Comment by LaurenBuchman** [comment]
Hey @mariafciampa can you provide any updates on timelines for a project scope from the vendor in the project board?

**Comment by mariafciampa** [comment]
@LaurenBuchman Yes, I'm finalizing timelines with vendor this morning. They initially suggested a longer timeline, and since we've created the script, storyboard, and the bulk of the assets, I'm asking for a deadline of June 16/17. As soon as they confirm that date works for their team, I'll add that date to the project board.

**Comment by LaurenBuchman** [comment]
Great, thanks!
